src/plugins/data_table/data_table.js

```diff
@@ -262,14 +282,15 @@ var LocalDataTable = (function() {
             if (this.isNontotalsColumn && this.isNontotalsColumn(col)) {
               // If column is a non totals column, draw "Grand totals" on the first one and the rest are empty
               if (hasGrandTotalsCell) {
-                $grandTotalsRow.append($("<td></td>"));
+                $grandTotalsRow.append($("<td>"));
               } else {
                 hasGrandTotalsCell = true;
-                $grandTotalsRow.append($("<td>Grand Total</td>"));
+                var grandTotalsCellText = this.totalsRow.hasUniques ? "Total Gross<br>Total Net" : "Grand Total";
```

Comment:
not a huge fan this logic is in backdraft at all... probably fine for now but better would be to pass in a grand total label renderer (it could ask about uniques and then add the `br` etc).  just a thought